### PR TITLE
Deprecate Ruby 2.6

### PR DESCRIPTION
**Pull request by JoeSouthan**
- Update dependencies
- Deprecate Ruby 2.6

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
   validate:
     strategy:
       matrix:
-        ruby-version: ["2.6", "2.7", "3.0", "3.1", "3.2"]
+        ruby-version: ["2.7", "3.0", "3.1", "3.2"]
```
Comment (kevinrobayna) [inline .github/workflows/tests.yml:10]: 2.7 is also out of live right?

Comment (JoeSouthan) [reply .github/workflows/tests.yml:10]: Yeah but this should still work with 2.7

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 2.0.0 - Sept 4, 2023
```
Comment (kevinrobayna) [inline CHANGELOG.md:1]: is this actually a major version? I'd consider it a minor since it's expected for everyone to have moved on from this version

Comment (JoeSouthan) [reply CHANGELOG.md:1]: Hmm you're right this would be a minor

**Review by kevinrobayna — APPROVED** [review]

**Pull request merged**